docs/compatibility.md

```diff
@@ -1,263 +1,187 @@
+|   - options                  |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - mac_address                |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - mem_limit                  |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - mem_swappiness             |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - memswap_limit              |     |     |     |     |     |     |     |     |  C  |  C  |  C  |  C  |  C  |
+| - network_mode               |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - networks                   |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+|   - <name>                   |  CS |  CS |  CS |  CS |     |     |     |     |     |     |     |     |     |
+|     - aliases                |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+|     - ipv4_address           |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+|     - ipv6_address           |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+|     - link_local_ips         |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |     |
+|     - priority               |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |
+| - pid                        |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - ports                      |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|   - mode:                    |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|     - host                   | CSk | CSk | CSk | CSk | CSk | CSk | CSk | CSk |  CS |  CS |  CS |  CS |  CS |
+|     - ingress                | CSk | CSk | CSk | CSk | CSk | CSk | CSk | CSk |  CS |  CS |  CS |  CS |  CS |
+|   - protocol                 |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|   - published                |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+|   - target                   |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - privileged                 |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  CK |  CK |
+| - read_only                  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
+| - restart                    |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |  C  |
+| - runtime                    |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |  CS |     |     |     |
+| - secrets                    |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |  *  |
```

Comment:
For secrets, `uid`, `gid` and `mode` are not support for Compose. Additionally, Compose does not support secrets declared external.

Reply:
Fixed.